package.json

```diff
@@ -98,7 +98,7 @@
     "@hot-loader/react-dom": "^16.13",
```

Comment:
What's the empty `git` file at the root directory for?

Reply:
Removed.